### PR TITLE
fix: change boolean check for conditional enabling file attachment

### PR DESCRIPTION
**Pull request by muneebGH**

### CODE REVIEW

File: .env.development [hunk 1]
```diff
@@ -32,4 +32,4 @@ FEATURE_BULK_ENROLLMENT=true
 HOTJAR_APP_ID=''
 HOTJAR_VERSION=6
 HOTJAR_DEBUG=''
-FEATURE_FILE_ATTACHMENT=false
+FEATURE_FILE_ATTACHMENT=true
```
Comment (sameenfatima78) [inline .env.development:35]: Are we enabling this feature for production too?

**Review by sameenfatima78 — APPROVED** [review]
Just one question, otherwise good to go

File: .env.development (hunk 1 above)
Comment (muneebGH) [reply .env.development:35]: not yet, once i test on stage, i will create edx-internal PR to enable it on production.

Comment (sameenfatima78) [reply .env.development:35]: Sounds good

**Pull request merged**